plugins/rss/start.sh

```diff
@@ -28,6 +28,12 @@ else
   else
     sed -i "s#regexp: placeholder#regexp: $rss_regex_esc#" /etc/transmission-rss.conf
   fi
+  if [[ $TRANSMISSION_RPC_ENABLED == 'true' ]] && [ -z $(grep login /etc/transmission-rss.conf) ]; then
```

Comment:
Should `TRANSMISSION_RPC_ENABLED` maybe be `TRANSMISSION_RPC_AUTHENTICATION_REQUIRED`?

That being said, we know we're contacting Transmission. And over RPC, right? There is no other way. So maybe the switch should just be `RPC_AUTH_ENABLED` and `RPC_USERNAME` + `RPC_PASSWORD`?

Reply:
Or something like that.

Reply:
Hmm, I was checking inside my container and only had the TRANSMISSION_RPC_ENABLED
Available so that’s why I used it but feel free to verify and I can fix